### ISSUE TITLE
Unable to use this addon reliably on portable NVDA's

### ISSUE THREAD

**Issue by serrebi** [fastfinge/eloquence_64#43]
Hello:
On NVDA snapshot, I can use this addon, if I install it into an installed NVDA, but if I try to make a portable NVDA and copy my current configuration, eloquence64 will not work.
It's easy to reproduce, just make a portable NVDA, and check the checkbox to copy your current config to the portable version. Then try to run the portable on another computer. Eloquence will not run.
Just thought I would open an issue for it, hopefully I'm not missing something obvious.
Brandon
